diff -r -c a/compiler-rt/lib/fuzzer/FuzzerLoop.cpp b/compiler-rt/lib/fuzzer/FuzzerLoop.cpp
*** a/compiler-rt/lib/fuzzer/FuzzerLoop.cpp    2021-01-05 09:42:57.571444091 +0800
--- b/compiler-rt/lib/fuzzer/FuzzerLoop.cpp    2021-01-05 09:46:56.377215961 +0800
***************
*** 366,371 ****
--- 366,372 ----
    Printf("stat::new_units_added:          %zd\n", NumberOfNewUnitsAdded);
    Printf("stat::slowest_unit_time_sec:    %zd\n", TimeOfLongestUnitInSeconds);
    Printf("stat::peak_rss_mb:              %zd\n", GetPeakRSSMb());
+   Printf("stat::total_run_time:           %zd\n", secondsSinceProcessStartUp());
  }

  void Fuzzer::SetMaxInputLen(size_t MaxInputLen)